webpack.config.base.js

```diff
@@ -1,18 +1,35 @@
 'use strict';
 
+var _ = require('lodash');
+var path = require('path');
 var webpack = require('webpack');
 
 module.exports = {
   module: {
-    loaders: [
-      { test: /\.js$/, loaders: ['babel-loader'], exclude: /node_modules/ }
-    ]
+    loaders: [{
+      test: /\.js$/,
+      loaders: ['babel-loader'],
+      include: [
+        path.resolve(__dirname, 'src'),
+        path.resolve(__dirname, 'node_modules', 'lodash')
+      ]
+    }]
+  },
+  node: {
+    process: false
   },
   output: {
     library: 'Redux',
     libraryTarget: 'umd'
   },
-  resolve: {
-    extensions: ['', '.js']
-  }
+  plugins: [
+    ['lodash/internal/baseIteratee.js', './toFunction.js'],
+    ['lodash/object/keys.js',           '../internal/baseKeys.js'],
+    ['lodash/object/keysIn.js',         '../internal/baseKeysIn.js']
+  ].map(function(pair) {
+    var resourceRegExp = RegExp('\\b' + _.escapeRegExp(pair[0]) + '$');
+    return new webpack.NormalModuleReplacementPlugin(resourceRegExp, pair[1]);
```

Comment:
For info on why `NormalModuleReplacementPlugin` instead of `resolve.alias` see this thead https://github.com/jdalton/redux/commit/7b50f3b7af340ab18663de4ca240a912c6d0a303#commitcomment-12833016